osclib/accept_command.py

```diff
@@ -14,7 +14,40 @@ def __init__(self, api):
         self.api = api
 
     def find_new_requests(self, project):
-        query = "match=state/@name='new'+and+(action/target/@project='{}'+and+action/@type='submit')".format(project)
+        query = "match=state/@name='new'+and+action/target/@project='{}'".format(project)
```

Comment:
This will now include things like `change_devel` requests. Are you sure the rest of the code can handle them properly or should it explicitly looks for submit or delete requests?

I had to restricted explicitly for SLE in #1121 which is the equivalent of `change_devel` in Factory so it be good to make sure we do not introduce another case.

Reply:
any kind of request must be reviewed by human and bot, if it's state is `new` then means all reviews are passed, in the past, it was merged manually(either change_devel request to Factory or add_role to SLE), this change just auto-merge those kind of request while executing accept_other_new.

Reply:
I understand they were reviewed, a question of can the code handle the structure of those different types of requests or does it make assumptions as the code I had to restrict did.

Is this needed for virtual delete requests? I do not remember this change in original PR and seems like a different topic entirely.